docs/application/web/guides/data/file-system.md

```diff
@@ -108,149 +107,261 @@ To manage file storages:
    ```
 
-2. Perform all actual operations, such as reading, writing, or closing, on the file through the `FileStream` object based on a position attribute, which represents the current position in the file:`function onOpenSuccess(fs) {    /* Write HelloWorld to the file */    fs.write('HelloWorld');    /* Move pointer to the beginning */    fs.position = 0;    /* Read the file content from the beginning */    fs.read(testFile.fileSize);    /* Close the file */    fs.close();}`
+   - use writeBlob() or writeBlobNonBlocking() to write `Blob` object contents to file.
+   ```
+   var blobToWrite = new Blob(['example blob content']);
+   fileHandleWrite.writeBlob(blobToWrite);
+   ```
 
-3. To copy a file or directory, use the `copyTo()` method. The following example copies the files to the `images/backup/` directory. Since the third parameter is set to `true`, any existing files with the same name in the target directory are overwritten.
+   - when file is no longer needed, remember to close it
+   ```
+   fileHandleRead.close();
+   ```
+ - You can copy, move and rename files and directories within the file system with the `copyFile()`, `copyDirectory()`, `moveFile()`, `moveDirectory()` and `rename()` methods. During copy or move operations, if a file or directory of the same name already exists in the target location, the overwrite input parameter of the method defines whether the existing file is overwritten.
+
+1. To copy a file, use the `copyFile()` method. The following example copies the file to the `images` directory. Since the third parameter is set to `true`, any existing file or directory with the same name in the target directory is overwritten.
 
    ```
-   var files; /* Assume that this is an array of File objects */
+   var filePathAndName = 'documents/exampleFile.jpg';
+   var destination = 'images/copyOfExampleFile.jpg';
    function onSuccess() {
        console.log('success');
    }
+   var errorCallback = function(error) {
+       console.log(error);
+   }
+   tizen.filesystem.copyFile(filePathAndName, destination, true, onSuccess, errorCallback);
+   ```
+
+2. To copy a directory, use the `copyDirectory()` method. The following example copies the directory to the `documents/` directory. Since the third parameter is set to `true`, any existing file or directory with the same name in the target directory is overwritten.
+
+   ```
+   var directoryPathAndName = 'downloads/exampleDirectory';
+   var destination = 'documents/copyOfExampleDirectory';
+   function onSuccess() {
+       console.log('success');
+   }
+   var errorCallback = function(error) {
+       console.log(error);
+   }
+   tizen.filesystem.copyDirectory(directoryPathAndName, destination, true, onSuccess, errorCallback);
+   ```
+
+3. To move a file, use the `moveFile()` method. The following example moves the files to the `music/` directory. Since the third parameter is set to `false`, no existing files with the same name in the target directory are overwritten.
```

Comment:
directory is overwritten:

Use a ":" at the end of the sentence as we have used the word following.

Reply:
Done.